deploy/charts/README.md

```diff
@@ -0,0 +1,30 @@
+# Helm charts
+
+Each folder of this directory contains a single [helm chart](https://helm.sh/docs/topics/charts/).
+
+A single helm chart organizes a collection of templatized k8s yaml files.
+
+## Installing a Helm chart
+
+Installing a Helm chart means filling in the templates with a set of values and 
+applying the resources to a live cluster.
+
+Note: Helm uses k8s config for authentication. You can visit the GCP UI and click "CONNECT" on your cluster's page to get the command to configure the k8s config. Check if the k8s config points to the right cluster with `kubectl config current-context`.
```

Comment:
nit: include a sample screenshot?

Reply:
- sample output would be great too (i.e. an example of how the config points to the right cluster)
- please add a link to k8s config (is this the right link: https://kubernetes.io/docs/concepts/configuration/configmap/)

Reply:
Done.